docs/getting-started/create-an-app.md

```diff
@@ -0,0 +1,81 @@
+# Backstage App
+
+If you want to get set up quickly with your own Backstage project you can create a Backstage App.
+
+A Backstage App is a monorepo setup with `lerna` that includes everything you need to run Backstage in your own environment.
+
+## Create an app
+
+If you have [`npx`](https://github.com/npm/npx) installed simply run:
+
+```bash
+npx @backstage/cli create-app
+```
+
+Or with `yarn`:
+
+```bash
+yarn add --dev @backstage/cli
+yarn run backstage-cli create-app
```

Comment:
Not sure we want to recommend this. The directory structure you end up with doesn't make much sense imo. cwd/node_modules + cwd/package.json + cwd/app/node_modules + cwd/app/package.json + cwd/app/packages/app/package.json, etc